src/FormLabel.js

```diff
@@ -24,6 +24,13 @@ const propTypes = {
    */
   column: PropTypes.bool,
 
+  /**
+   * Label size variants. Only applicable if column is true
+   *
+   * @type {('sm'|'lg')}
+   */
+  size: PropTypes.string,
```

Comment:
This should be merged with the `column` prop.

Reply:
as in `column` should take `true` or `"sm"` or `"lg"`

Reply:
I see. But I thought it would be more consistent with <FormControl> this way?

Reply:
the issue is that the prop doesn't do anything if you're not specifying column...